server/src/main/java/org/elasticsearch/index/query/IntervalsSourceProvider.java

```diff
@@ -114,6 +120,12 @@ public Match(StreamInput in) throws IOException {
             this.ordered = in.readBoolean();
             this.analyzer = in.readOptionalString();
             this.filter = in.readOptionalWriteable(IntervalFilter::new);
+            if (in.getVersion().onOrAfter(Version.V_7_1_0)) {
+                this.useField = in.readOptionalString();
+            }
+            else {
```

Comment:
nit: you don't need the `else` ?

Reply:
useField is final, so it needs to be set somewhere

Reply:
I missed the `final`, thanks